frontend/src/index.tsx

```diff
@@ -36,9 +40,34 @@ declare global {
 (async () => {
   window.deckyAuthToken = await fetch('http://127.0.0.1:1337/auth/token').then((r) => r.text());
 
+  i18n
+    .use(Backend)
+    .use(LanguageDetector)
+    .use(initReactI18next)
+    .init({
+      load: 'currentOnly',
+      detection: {
+        order: ['querystring', 'navigator'],
+        lookupQuerystring: 'lng',
+      },
+      //debug: true,
+      fallbackLng: 'en-US',
+      interpolation: {
+        escapeValue: false,
```

Comment:
what's this do?

Reply:
How i18next should handle tag injection inside their properties, see here:

https://www.i18next.com/translation-function/interpolation#unescape

Reply:
IMO, leave escaping enabled as it will surely cause later headache for us leaving it off. For long, formatted documents like the store About page, it might be better to convert those to Markdown and use the MarkdownRenderer (and also move that component to not be specialized for the patch notes).

Reply:
Reenabled the manual escape as requested.